projects/igniteui-angular/src/lib/grids/grid/grid.component.html

```diff
@@ -32,13 +32,13 @@
                 <igx-icon [style.visibility]="(!last || dropAreaVisible) ? 'visible' : 'hidden'" >arrow_forward</igx-icon>
             </span>
         </ng-container>
-        <div igxGroupAreaDrop [style.visibility]="dropAreaVisible ? 'visible' : 'hidden'" [class]="groupAreaHostClass">
+        <div igxGroupAreaDrop [style.visibility]="dropAreaVisible ? 'visible' : 'hidden'" [class]="groupAreaHostClass" [attr.gridId]='this.id'>
             <ng-container *ngTemplateOutlet="dropAreaTemplateResolved"></ng-container>
         </div>
     </igx-chips-area>
 </div>
 
-<div class="igx-grid__thead">
+<div class="igx-grid__thead" [style.width.px]='outerWidth'>
```

Comment:
It is the calculated width (in px)  + scrollBarWidth (if there is scorlbar). It is applied because IE cannot size things correctly without explicit width being set.

Reply:
I tried to reproduce this in a sample we have and I can't. Can you clarify in what scenarios this occurs?

![image](https://user-images.githubusercontent.com/1472513/52352575-871d7a00-2a35-11e9-8a0a-4176a94a519d.png)

Reply:
I don't think the changes with the external scrollbar are on live yet. You can reproduce it by getting master, run the dev samples and open a sample with a grid with width on IE, for example the "Grid Cell Editing" sample: 
![image](https://user-images.githubusercontent.com/10397980/52353175-b08ad580-2a36-11e9-94c1-d1cdaae7f1c3.png)

Reply:
Adding explicit pixels to every single part of every type of grid is definitely not the only solution, and probably the least desired solution for me. This affects every browser, not just IE, so I cannot accept this solution.

Reply:
Ok. In that case the change will be reverted and separate issue will be logged to be resolved at a later time.